packages/airnode-adapter/src/types.ts

```diff
@@ -49,18 +50,21 @@ export interface Config {
 
 export type ValueType = string | BigNumber | boolean | Array<ValueType>;
 
-export const solidityBaseTypes = ['uint256', 'int256', 'bool', 'bytes32', 'address', 'bytes', 'string'] as const;
-export const artificialTypes = ['string32'] as const;
-export const baseResponseTypes = [...solidityBaseTypes, ...artificialTypes] as const;
 export type BaseResponseType = typeof baseResponseTypes[number];
 // Use might pass a complex type (e.g. int256[3][]) which we cannot type
 export type ResponseType = string;
 
 export interface ReservedParameters {
-  readonly _path?: string;
-  readonly _times?: string | BigNumber;
-  readonly _type: ResponseType;
-  readonly _relay_metadata?: string;
+  _path?: string;
+  _times?: string;
```

Comment:
I am not sure why this function accepted `BigNumber` - I think the only valid value is `string` (also there were no issues when I removed BigNumber from the type).

Reply:
I couldn't recall but this is probably important

Reply:
@andreogle do you remember?

Reply:
I can't quite remember either, sorry. Should be fine to remove it then